src/map/common/lapacksrc/scripts/regen-files.sh

```diff
@@ -26,12 +30,11 @@ usage() {
 clean() {
 
     rm -f $script_dir/../netlib/._*
-    rm -f $script_dir/../netlib/*.[fF]
```

Comment:
same?

Reply:
Yes, will remove.

Reply:
No, [] is regenex, while {} runs each of those.

So this:
rm -f $script_dir/../netlib/*.{f,F,f90,F90}

Translates to this:
rm -f $script_dir/../netlib/*.f
rm -f $script_dir/../netlib/*.F
rm -f $script_dir/../netlib/*.f90
rm -f $script_dir/../netlib/*.F90

Reply:
While [fF] would translate to a single char between the ranges of 'f to f' or 'F to F'

Reply:
However, end results here are the same.